index.html

```diff
@@ -29,82 +26,123 @@ <h1>Straight to the Point Archery</h1>
-            to the professional archer. Once set up with proper fitting equipment, each archer will be given the basic
-            safety instruction to start and will be introduced to various shooting techniques.
-        </p>
-
-        <p>
-            From Beginner to Elite, archers of all ages and ability learn about this exciting Olympic and international
-            sport. Sign up for a group lesson, private class, schedule a birthday or corporate party or compete at one
-            of our many tournaments. Straight to the Point Archery also offers homeschooling programs, kids club events,
-            YMCA classes, Boy and Girl Scouts achievement programs and several other youth group activities. We are
-            dedicated to teaching valuable life skills to the youth in our community through the wonderful discipline of
-            archery.
-        </p>
-
-        <p>
-            Straight to the Point Archery has instructors that are qualified level 1 and level 2 instructors and have
-            met the safety requirements of the USA Archery (USAA). We teach in a positive learning environment. Our
-            instructors also have extensive knowledge of the most advanced archery equipment and all forms of archery
-            from the traditional long bow to the modern compound. Straight to the Point Archery also has specific
-            instructors for recreational archery, competitive archery and bow hunting.
-        </p>
-
-        <p>
-            Straight to the Point Archery has an excellent network of contacts with the USA Archery (USAA), Washington
-            State Archery Association (WSAA), Washington State Bow Hunters (WSB), National Field Archery Association
-            (NFAA) and many more national and state archery associations. These partnerships allow you to feel confident
-            in our facility and staff.
-        </p>
-
-        <p>
-            Straight to the Point Archery’s staff has decades of experience in the archery industry. We are committed to
-            educating you with any questions that you may have.
-        </p>
+        <h1>Straight to the Point Archery</h1>
+
+        <section class="about">
+            <div>
+                <h2>About Us</h2>
+            </div>
+            <div class="more center-button">
+                <p>Straight to the Point Archery is a non profit archery education center founded in 2005 by Giap
+                    Nguyen.
+                    Our
+                    state of the art educational and training facility incorporates detailed technical video analysis
+                    combined
+                    with generations of archery experience from certified coaches and instructors.
+                </p>
+            </div>
+        </section>
+
+        <div class="home">
+            <section class="facilities">
+                <h2>Facilities</h2>
+
+                <img src="images/facility.jpeg">
```

Comment:
Remember that all images need an alt attribute. It's especially important for accessibility.